gcs_oauth2_boto_plugin/oauth2_helper.py

```diff
@@ -100,6 +100,10 @@ def OAuth2ClientFromBotoConfig(config,
     client_secret = config.get('OAuth2', 'client_secret',
                                os.environ.get('OAUTH2_CLIENT_SECRET',
                                               CLIENT_SECRET))
+    ca_certs_file=config.get_value('Boto', 'ca_certificates_file')
+    if ca_certs_file == 'system':
+      ca_certs_file = None
```

Comment:
Can you describe the use case where we'd want to fall through to the system certificates file?

In general I'm more in favor of explicit configuration, especially where security is involved.